projects/schematics/src/migrations/3_0/constructor-deprecations/data/checkout-config.service.migration.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  CHECKOUT_CONFIG,
+  CHECKOUT_CONFIG_SERVICE,
+  ROUTING_CONFIG_SERVICE,
+  SPARTACUS_CORE,
+  SPARTACUS_STOREFRONTLIB,
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const CHECKOUT_CONFIG_SERVICE_MIGRATION: ConstructorDeprecation = {
```

Comment:
delete this file and from constructor-deprecations-data as the constructor is still the same